src/main/java/com/carina/methods/saucedemo/android/pages/HamburgerMenuScreen.java

```diff
@@ -0,0 +1,28 @@
+package com.carina.methods.saucedemo.android.pages;
+
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = com.carina.methods.saucedemo.commons.pages.HamburgerMenuScreenBase.class)
+public class HamburgerMenuScreen extends com.carina.methods.saucedemo.commons.pages.HamburgerMenuScreenBase {
```

Comment:
please check all classes and check why it shows full path
```suggestion
@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = HamburgerMenuScreenBase.class)
public class HamburgerMenuScreen extends HamburgerMenuScreenBase {
```